docs/applications/cli.md

```diff
@@ -35,4 +35,27 @@ Usage: bitcoin-s-cli [options] [<cmd>]
 ```
 
 
-Now you are are ready to start the server that the cli sends commands to. Take a look at our [server](server.md) documentation on how to build and start the server.
\ No newline at end of file
+Now you are are ready to start the server that the cli sends commands to. Take a look at our [server](server.md) documentation on how to build and start the server.
+
+### Native binaries
+
+Bitcoin-s also supports building native executables for various platforms supported by the [graalvm native image](https://www.graalvm.org/reference-manual/native-image/) tool.
+
+You can build by using the [native image plugin for sbt](https://github.com/scalameta/sbt-native-image). This will download the appropriate graalvm
+version and the native image tool, and then build the cli
+
+```bashrc
+sbt cli/nativeImage
+```
+
+After the build is done, you can find the artifact here locally
+
+```bashrc
+app/cli/target/native-image/bitcoin-s-cli
+```
+
+We also publish native image binaries every time we merge a commit to master on github.
+As an example, you can [see the artifacts](https://github.com/bitcoin-s/bitcoin-s/runs/1792909760)
+in the upper right hand corner.
+
+If you don't want to build the `bitcoin-s-cli` yourself, you can download it for your platform there.
```

Comment:
Can link to all the latest runs here:

https://github.com/bitcoin-s/bitcoin-s/actions?query=workflow%3A%22Native+Image+bitcoin-s-cli%22